deploy_stable.bat

```diff
@@ -0,0 +1,2 @@
+git switch main
+gradlew deploy
```

Comment:
Don't you need a build as well?

Reply:
Deploy does a build and then a copy (I believe) so this should work.

Reply:
Make sure it does

Reply:
you can also use "@echo off" to prevent commands you're running from being displayed in your terminal

Reply:
@Irishumanoid You want to display them in case the build fails or it fails to deploy.

This also won't work since the `gradlew` used WPILib VSCode depends on environment variables like `JAVA_HOME` being set. I advise against setting them system-wide as the location changes year to year. There is a way to set environments for one specific command.

In bash it's:

```bash
export VAR="value"
program_to_run
```

In a Windows batch script it's:

```batch
set "var=value"
program_to_run
```

In this case you need to set `JAVA_HOME` to the WPILib Java installation, which resides in `C:\Users\Public\wpilib\[YEAR]\jdk` before you invoke `gradlew`.

You additionally should keep the terminal open in case something happens. You'll want to invoke the `gradlew deploy` command with cmd.

To do this, you can do `call gradlew deploy`

This will output the command to a Command Prompt window; however, it won't keep it open. The window will close as soon as the command is finished (not what we want!)

Windows conveniently has a batch command which will pause until a key is pressed called `pause` which will do this.